grpc/tests/src/test/step.rs

```diff
@@ -9,21 +9,21 @@ use casper_execution_engine::{
 use casper_types::{
     account::AccountHash,
     auction::{
-        BidPurses, Bids, SeigniorageRecipientsSnapshot, BIDS_KEY, BID_PURSES_KEY,
+        BidPurses, Bids, SeigniorageRecipientsSnapshot, BIDS_KEY, BID_PURSES_KEY, BLOCK_REWARD,
         SEIGNIORAGE_RECIPIENTS_SNAPSHOT_KEY, VALIDATOR_REWARD_PURSE,
     },
     ContractHash, Key, ProtocolVersion, PublicKey,
 };
 
 const ACCOUNT_1_PK: PublicKey = PublicKey::Ed25519([200; 32]);
 const ACCOUNT_1_ADDR: AccountHash = AccountHash::new([200; 32]);
-const ACCOUNT_1_BALANCE: u64 = 10_000_000;
-const ACCOUNT_1_BOND: u64 = 100_000;
+const ACCOUNT_1_BALANCE: u64 = 10_000_000_000_000;
```

Comment:
Probably don't _need_ to bump these now.  In fact, I'd like to make sure that the only problem before was indeed the `BLOCK_REWARD` one...

Reply:
did this